.prettierrc

```diff
@@ -1,7 +1,8 @@
 {
+  "arrowParens": "avoid",
```

Comment:
I added this to conform with the existing snippet conventions in this repo. The vast majority don’t use parens around arguments to arrow functions, so this was least disruptive.